packages/bits/demo/src/components/demo/textbox/textbox-visual-test/textbox-visual-test.component.ts

```diff
@@ -8,6 +8,9 @@ import { Component } from "@angular/core";
 export class TextboxVisualTestComponent {
     public isRequired = true;
     public value = "";
+    public textExample: string = 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquid dolores ' +
+        'ex harum itaque laborum laudantium quae quo, soluta suscipit voluptate. Accusamus aliquid amet architecto, ' +
+        'doloremque ducimus eveniet facere facilis fugit itaque';
```

Comment:
You can use backticks for multiline strings, so you don't have to concatenate them.